cmd/ocsp-responder/main.go

```diff
@@ -55,34 +55,38 @@ serialNumber field, since we will always query on it.
 
 */
 type DBSource struct {
-	dbMap     *gorp.DbMap
+	dbMap     dbMapInterface
 	caKeyHash []byte
+	log       *blog.AuditLogger
+}
+
+type dbMapInterface interface {
+	SelectOne(holder interface{}, query string, args ...interface{}) error
+	Insert(list ...interface{}) error
```

Comment:
Because Insert is only used by the test, it doesn't have to be part of the interface used by the live code. Remove it and call the interface dbSelectable or dbSelector or something. Any opinions on naming idioms here @jmhodges?